models/models.py

```diff
@@ -1,27 +1,12 @@
 from models.processing import filter_dataset, load_dataset
 from dataclasses import dataclass
 from typing import List
+from datetime import datetime
+import pandas as pd
+from gurobipy import *
 
 
-@dataclass
```

Comment:
We should keep those classes and populate them at the end of the pareto route function so that we can easily access the lements we need (name, coordinates, rating, etc.) in the web app.